app/models/bulk_tagging_source.rb

```diff
@@ -0,0 +1,19 @@
+class BulkTaggingSource
+  def source_name_to_content_key_map
+    {
+      document_collection: 'documents',
+      topic: 'children',
+      mainstream_browse_page: 'children',
```

Comment:
These values got me a bit confused.  I thought we would access links by `topic` and `mainstream_browse_page` rather than `children`.